wagtailmodelchoosers/client/components/BaseChooser.js

```diff
@@ -186,7 +188,10 @@ class BaseChooser extends React.Component {
             url={initialUrl}
             onClose={this.onClose}
             onSelect={this.onSelect}
-            {...this.props}
+            label={label}
+            endpoint={endpoint}
+            filters={filters}
+            list_display={listDisplay}
```

Comment:
As far as I can tell, it's missing `pk_name`, `page_size`, `page_size_param` and `translations`.